package.json

```diff
@@ -22,14 +22,18 @@
   "homepage": "https://github.com/HackGT/buzzer#readme",
   "private": true,
   "dependencies": {
+    "@types/graphql": "^0.12.4",
```

Comment:
@types dependencies should go in `devDependencies`. You can do this by passing the `---save-dev` flag when you `npm install`